.github/workflows/python-runtests-basic.yml

```diff
@@ -27,7 +27,7 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install flake8 pytest
-        if [ -f requirements_dev.txt ]; then pip install -r requirements_dev.txt; fi
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
     - name: Test with pytest
       run: |
-        python check_cloudside.py --runslow
+        pytest -x
```

Comment:
The `-x` flag here will stop the test on the first failure. Is that what you'd like?

Reply:
no, you're right it's better to get a full list of test failures from CI. I switched this to -v instead.